### ISSUE TITLE
CommonClient.default_log fails with KeyError if LogEntry doesn't contain a 'msg' Child

### ISSUE THREAD

**Issue by talex24** [dsoprea/PySvn#17]
The current implementation of 'CommonClient.default_log' assumes that each 'logentry' always contains 'msg', 'author', and 'date' Elements in the XML output of svn log command.

I do have access to an svn repository where some few revisions don't contain 'msg' Elements.
(Perhaps this repository is corrupted, but I don't have administrator access to the svn repositoty to check it.)

It would be helpfull to provide default value '' to all fields of 'entry_info' in order to prevent a KeyError in
common.py, line  203 in such situation.

**Comment by ardumont** [comment]
Hello, i concur with @talex24's analysis.

Here is an example repository http://example.googlecode.com/svn/.
The author is not provided at revision 1.

Some more references:
- http://svnbook.red-bean.com/en/1.7/svn-book.html#svn.ref.properties

```
svn:

These are the versioned properties that Subversion reserves for its own use:

...

svn:author

    If present, contains the authenticated username of the person who created the revision. (If not present, the revision was committed anonymously.)
```

- http://stackoverflow.com/questions/20252633/subversion-no-author-date-and-message#20256713

Is there some blocking point from the provided PR?
(Maybe returning None might be better than the empty string?)

Cheers,

**Comment by dsoprea** [comment]
Resolved via #13. Thanks, guys.
